doc/nrf/ug_thingy91.rst

```diff
@@ -24,10 +24,18 @@ This guide gives you more information on the various aspects of Thingy:91.
 Connecting to Thingy:91
 ***********************
 
-For connecting to Thingy:91, you can use `LTE Link Monitor`_, `Trace Collector`_, or a serial terminal.
+You can connect to Thingy:91 wirelessly (using the `nRF Toolbox`_ app) or over a serial connection (using `LTE Link Monitor`_, `Trace Collector`_, or a serial terminal).
 
-Using serial ports
-==================
+Using nRF Toolbox
+=================
+
+To connect to your Thingy:91 wirelessly, you need to meet the following prerequisites:
+
+* The :ref:`connectivity_bridge` installed on your Thingy:91.
+* The :ref:`nus_service_readme` enabled.
```

Comment:
What do you mean by having nus enabled? It seems to be enabled in the connectivity app project conf file.

Reply:
The connectivity bridge allows the NUS to be enabled or disabled by modifying a `Config.txt` on the mass storage device. This line should probably point to those instructions

Reply:
Where do we want to point to here?

Reply:
When I first read it, it means to me that I need to switch the Kconfig option. So I agree, that we can link here a link or reference to doc describing how to enable NUS, of course if such doc exists.

Reply:
The Overview part of the connectivity_bridge application seems to be a good candidate but you must put there label to have possibility to reference it here

Reply:
Lets try with note :)